Jolyn

wanderess opened the pull request:
Hello,

Just created a pull request in case the changes aren't reflected in my branch:
- Added the navbar, slideshow, and social media bar into all pages
- Edited the "About Us" page

wanderess committed:
Added the links into navbar and social media bar

wanderess committed:
Added navbar, slideshow, and social media bar into all pages

wanderess committed:
added a jumbotron on the homepage

bkotryna committed:
Merge branch 'master' into Jolyn

bkotryna commented:
There were conflicts with the master branch, which I tried to sort out... Not sure about the outcome.

wanderess committed:
Resizing and some styling

- Changed font and color of jumbotron text
- Made all elements (navbar, images, form, and maps) responsive to
resizing
- Resized picture slideshow
- Added border to picture slideshow
- Added two images to Bath webpage
- Made first sentence of every div stylised

wanderess committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Jolyn

# Conflicts:
#	about_x1.html
#	bath_x1.html
#	brighton_x1.html
#	css/jolyn.css
#	edinburgh_x1.html
#	home_x1.html
